ql/src/java/org/apache/hadoop/hive/ql/exec/repl/bootstrap/load/table/LoadTable.java

```diff
@@ -159,10 +159,20 @@ public TaskTracker tasks() throws Exception {
     return tracker;
   }
 
-  private ReplLoadOpType getLoadTableType(Table table) throws InvalidOperationException, HiveException {
+  private ReplLoadOpType getLoadTableType(Table table, boolean isBootstrapDuringInc)
+          throws InvalidOperationException, HiveException {
     if (table == null) {
       return ReplLoadOpType.LOAD_NEW;
     }
+
+    // In case user has asked for bootstrap of transactional table, we replace the old one if present. This is to
+    // make sure that the transactional info like write id etc for the table is consistent between the
+    // source and target cluster.
+    if (isBootstrapDuringInc && AcidUtils.isTransactionalTable(table)) {
```

Comment:
I think, this check should be done even for external tables. Why don't we always replace table in case of isBootstrapDuringInc?